test/test_common/utility.cc

```diff
@@ -80,6 +80,18 @@ std::string TestUtility::bufferToString(const Buffer::Instance& buffer) {
   return output;
 }
 
+void TestUtility::feedBufferWithRandomCharacters(Buffer::Instance& buffer, uint64_t n_bytes) {
+  std::string const chars = "abcdefghijklmnaoqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
+  std::random_device rd;
```

Comment:
This could be just a pseudo-random to avoid problems in case of low entropy in the system.